crates/bevy_math/src/direction.rs

```diff
@@ -229,7 +229,7 @@ impl Dir2 {
         self.rotation_from_x().inverse()
     }
 
-    /// Get the rotation that rotates this direction to the Y-axis.
+    /// Get the rotation that rotates the Y-axis to this direction.
```

Comment:
Question: would it be clearer and/or easier to remember if we used the term "from" in the docs?
```suggestion
    /// Get the rotation that rotates to this direction from the Y-axis.
```

Reply:
Good idea!

Reply:
I tried this out, but I found that it makes the wording a little weird between the `to/from` method docs, so I think I'm going to leave it alone for now.